docs/how-to-guides/how-to-test-with-selenium.md

```diff
@@ -88,8 +88,8 @@ Open your Properties in the Tests project. Go to Reference Paths, enter `C:\Sele
 
 BaseTest is a helper class that makes it easier to test multiple browsers. The source code is available:
 
-* BaseTest helper class [here](https://github.com/Starcounter/KitchenSink/blob/master/test/KitchenSink.Tests/Test/BaseTest.cs)
-* Using of BaseTest class [here](https://github.com/Starcounter/KitchenSink/blob/master/test/KitchenSink.Tests/Test/SectionBoolean/CheckboxPageTest.cs)
+* BaseTest helper class [here](https://github.com/Starcounter/UniformDocs/blob/master-2.4/test/UniformDocs.Tests/UniformDocs.Tests.csproj)
```

Comment:
This link is incorrect, it should go to https://github.com/Starcounter/UniformDocs/blob/master-2.4/test/UniformDocs.Tests/Test/BaseTest.cs